### PR TITLE
{AKS} Monitoring addon MSI auth - Remove unused ContainerInsights Solution resource

### PR DESCRIPTION
**Pull request by ganga1980**
**Related command**

**Description**

This PR removes the creation unused ContainerInsights Solution resource

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
AKS

**Comment by zhoxing-ms** [comment]
> ERROR: ************* Module azure.cli.command_modules.acs.addonconfiguration
src/azure-cli/azure/cli/command_modules/acs/addonconfiguration.py:5:0: W0611: Unused import datetime (unused-import)
src/azure-cli/azure/cli/command_modules/acs/addonconfiguration.py:5:0: W0611: Unused import datetime (unused-import)

@ganga1980 Please fix the CI issue

**Comment by ganga1980** [comment]
> > ERROR: ************* Module azure.cli.command_modules.acs.addonconfiguration
> > src/azure-cli/azure/cli/command_modules/acs/addonconfiguration.py:5:0: W0611: Unused import datetime (unused-import)
> > src/azure-cli/azure/cli/command_modules/acs/addonconfiguration.py:5:0: W0611: Unused import datetime (unused-import)
> 
> @ganga1980 Please fix the CI issue



> > ERROR: ************* Module azure.cli.command_modules.acs.addonconfiguration
> > src/azure-cli/azure/cli/command_modules/acs/addonconfiguration.py:5:0: W0611: Unused import datetime (unused-import)
> > src/azure-cli/azure/cli/command_modules/acs/addonconfiguration.py:5:0: W0611: Unused import datetime (unused-import)
> 
> @ganga1980 Please fix the CI issue

>  @zhoxing-ms Fixed this issue already.

**Review by zhoxing-ms — APPROVED** [review]

File: src/azure-cli/azure/cli/command_modules/acs/addonconfiguration.py [hunk 1]
```diff
@@ -459,121 +458,6 @@ def ensure_container_insights_for_monitoring(
...
-        except HttpResponseError as ex:
-            if ex.status_code == 404:
-                is_exists = False
-                break
-            if retry_count >= (_MAX_RETRY_TIMES - 1):
-                raise ex
-    return is_exists
 
 
 def _invoke_deployment(
```
Comment (FumingZhang) [inline src/azure-cli/azure/cli/command_modules/acs/addonconfiguration.py:463]: I suppose after removing the `if not _is_container_insights_solution_exists` code block, `_invoke_deployment` is also not used anymore. Please help confirm and remove it if it's useless.

Comment (ganga1980) [reply src/azure-cli/azure/cli/command_modules/acs/addonconfiguration.py]: > `_invoke_deployment`

> Yes, this is unused also hence the removed this.

**Comment by ganga1980** [comment]
Hi, @zhoxing-ms , can you please help on merging this PR?

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**